src/app/cli/src/tests/coda_batch_payment_test.ml

```diff
@@ -25,6 +25,12 @@ let main () =
     Coda_worker_testnet.test logger num_nodes proposers snark_work_public_keys
       Cli_lib.Arg_type.Sequence ~max_concurrent_connections:None
   in
+  (* Wait for nodes to initialize *)
```

Comment:
See this duplicated in multiple places. Can we put this as a function somewhere, like in Transition_router?

Reply:
I think I should put inside `Coda_worker_testnet.test`.